mne/preprocessing/artifact_detection.py

```diff
@@ -4,14 +4,74 @@
 
 
 import numpy as np
-
+from scipy.stats import zscore
+from scipy.ndimage.measurements import label
```

Comment:
use nested imports for these scipy modules.
The avoids loading all scipy with importing mne which makes it slow